### DISCUSSION TITLE
crowdsourced resolver

### DISCUSSION THREAD

**Discussion by ghost** [Fedoraware/Fedoraware#1207] [Ideas]
what if we like, calculated the lag comp that non cheating players are getting, and then like we look at where there bullets go and if it hits a player we are tryna resolve but that player doesn't take damage we would rotate their yaw or pitch or whatever, wouldn't that be cool , please like and share this idea if you think thats cool.

**Comment by Gergish** [comment]
what cool idea. but i dont know how to code :(

**Comment by BindUser** [comment]
But u gotta remember that fedoraware's aimbot isn't perfect so it will sometimes missed aka 0 damage
